quasar_site/src/pages/DataType.vue

```diff
@@ -57,13 +57,15 @@ import { DataTypes } from '../RhinoCommonApi'
 
 export default {
   data () {
+    const mostRecent = ViewModel.mostRecentSince().toFixed(1)
```

Comment:
`toFixed(1)` is converting a number to a formatted string. I don't think that is needed here as we can compare numbers directly.

Reply:
We can define new colors in quasar.variables.sass. I'm not sure 'warning' is the best variable to use here. How about a new latestVersion variable?

Reply:
We shouldn't have to test for `>` and only need to test for '==='. If member.since is greater than version, then we have a bug

Reply:
Defining new colours is more than just adding a new variable to quasar.variables.sass; `:color=something` expands to the class `text-something` which [needs to be defined somewhere](https://quasar.dev/style/color-palette#Adding-Your-Own-Colors). For now I switched it to use the `accent` colour. So much to learn!